src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -37,7 +37,9 @@ public partial class InfiniteScrollList : UserControl
         public delegate void UpdateButtonClickEventHandler(PackageItemListViewModel[] selectedPackages);
         public event UpdateButtonClickEventHandler UpdateButtonClicked;
 
-        // This exists only to facilitate unit testing.
+        /// <summary>
+        /// This exists only to facilitate unit testing. It is triggered after LoadItems() is just before finished
```

Comment:
It is triggered after LoadItems(), just before it is finished.

Reply:
Rephrased.